exec/cnex/nstring.c

```diff
@@ -414,7 +475,9 @@ char *to_string(char *dest, size_t len, uint32_t val)
 
 const char *string_ensureNullTerminated(TString *s)
 {
-    // This function ensures that the string is null terminated without adding to actual length of the string.
+    // This function ensures that the string is null terminated without adding to the actual length of the string,
+    // and returns the data member as a const char *.  The caller must NOT call free() on the return vlaue.  Instead
```

Comment:
You can't call `free()` on a `const char *` anyway, the compiler checks this for you. See https://godbolt.org/z/1G1sYrYeh Not sure why this comment needed to be changed.

Reply:
Reverted comment back to original.